v0/destinations/af/transform.js

```diff
@@ -18,7 +18,11 @@ const {
 } = require("./config");
 
 function responseBuilderSimple(payload, message, destination) {
-  const endpoint = ENDPOINT + message.context.app.namespace;
+  const endpoint =
```

Comment:
@ruchiramoitra We need to check for `device-type` and modify the endpoint accordingly.